app/src/main/java/co/develhope/chooseyourownbeer/BeerDetail.kt

```diff
@@ -11,28 +11,42 @@ import co.develhope.chooseyourownbeer.model.Beer
 
 class BeerDetail : Fragment() {
 
-    private var _binding: BeerDetailBinding?= null
+    private var _binding: BeerDetailBinding? = null
     private val binding get() = _binding!!
+    private var beerId: Int? = null
 
     override fun onCreateView(
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
         _binding = BeerDetailBinding.inflate(inflater, container, false)
-        val beerId= arguments?.getLong("BEER_ID") ?: 0
-        val beer = getBeerFromId(beerId)
-        getBeerFromId(beerId)?.let { setupUI(it) }
+        // val beerId = arguments?.getLong("BEER_ID") ?: 0
+        val beer = beerId?.let { getBeerFromId(it) }
+        beer?.let { setupUI(it) }
```

Comment:
Nel caso in cui l'oggetto non fosse trovato che succede? Direi di mostrare una textview con un messaggio di errore

Reply:
Anche un Toast andrebbe bene o consigli la TextView?